lib/inc/internal/facts/linux/os_cisco.hpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file
+ * Declares the Cisco Linux operating system query helper.
+ */
+#pragma once
+
+#include <internal/facts/linux/os_linux.hpp>
+
+namespace facter { namespace facts { namespace linux {
+
+    struct os_cisco : os_linux
```

Comment:
I forget, will this generate doxygen warnings for missing comments? I can't remember if we were limiting docs to the public headers or not.